### ISSUE TITLE
Bug: 21-4140 Confirmation page loses focus

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#122886]
# Bug Report

- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

[While doing accessibility testing for 21-4140](https://github.com/department-of-veterans-affairs/va.gov-team/issues/120468) I discovered that when a user navigates to the confirmation page from the review page, focus is *not* managed on the confirmation page. The page scrolls up to mid-form, and there is no focus indicator. When a user presses tab, focus does go to the link `Check the status of your form on My VA`.

## What I expected to happen

When the confirmation page loads, I expect focus and scroll to go to the h2 `Form submission started on $date`

## Reproducing

**URL(s) where this bug occurs**

- https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/confirmation

**Steps to reproduce:**

1. Fill out this form https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/
2. Click "Submit this application" on the review page
3. Observe the page scroll and loss of focus.

**Video**

https://github.com/user-attachments/assets/7c766b14-8469-49ab-b99c-dee859795512

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [X] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other
